bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/XmlItem.java

```diff
@@ -522,11 +524,13 @@ public Object copy(Map<Object, Object> refs) {
      */
     @Override
     public XmlValue getItem(int index) {
-        if (index != 0) {
+        if (index == 0) {
+            return this;
+        } else if (index > 0) {
```

Comment:
Can we get rid off the `else` here like this?
```suggestion
        } 
        
        if (index > 0) {
```

Reply:
I think it is better to use if and else if rather than using two ifs

Reply:
are you suggesting to remove `else if` and add `if`?.
I think using `if` and `else if` will be better rather than using to `if`s.

Reply:
updated. 5ff4ac52aa7ec017f6582c9daaf91208f3566c3c